caikit/runtime/servicers/global_train_servicer.py

```diff
@@ -115,31 +104,30 @@ def Train(self, request, context, *_, **__) -> TrainingJob:
 
         try:
             with alog.ContextLog(log.debug, outer_scope_name):
+                module = None
                 for mod in caikit.core.registries.module_registry().values():
                     if mod.TASK_CLASS:
                         train_request_for_mod = (
                             ModuleClassTrainRPC.module_class_to_req_name(mod)
                         )
                         if train_request_for_mod == desc_name:
-                            model = mod
+                            module = mod
```

Comment:
nit that totally doesn't have to be done now but the `ModuleClassTrainRPC` for this particular request is passed in as `caikit_rpc` so this entire block can be replaced with `module = caikit_rpc.module`

(I think it might actually be called `caikit_rpc.clz` right now though, which, 🤢 )

Reply:
Making this change breaks a lot of tests (though it seems to work in the end-to-end ones that aren't testing the function directly). I'll make an issue to clean it up later

Reply:
https://github.com/caikit/caikit/issues/326